samples/metadata-sample.c

```diff
@@ -70,29 +55,28 @@ int main()
     NVMfTransport *transport10 = create_nvmf_transport(type, address_family, address, "4429", "nqn.2020-11.com.github.hyperwarp:cnode10");
     PhysicalDisk *physical_disk10 = create_physical_disk(metadata, transport10, 266144ULL, 4096ULL, allocator);
 
-
     VirtualDisk *virtual_disk1 = create_virtual_disk(metadata, "vdisk1", 3ULL, allocator);
     VirtualDisk *virtual_disk2 = create_virtual_disk(metadata, "vdisk2", 3ULL, allocator);
 
     print_metadata(metadata, 0);
 
-    metadata_persist(database, metadata);
+    //metadata_persist(database, metadata);
     printf("Wrote MetaData to FDB\n");
 
     metadata__free_unpacked(metadata, NULL);
     allocator = NULL;
     free(allocator);
 
-    metadata = metadata_get(database);
+    //metadata = metadata_get(database);
```

Comment:
can you add the new way of loading metadata here and not just comment out the old way?